src/backend/joanie/core/api/admin.py

```diff
@@ -111,6 +114,19 @@ def get_queryset(self):
 
         return super().get_queryset()
 
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=[permissions.IsAuthenticated],
+    )
+    # pylint: disable=invalid-name
+    def me(self, request):
```

Comment:
Ditto for the method located at joanie/core/api/client.py : `def me(self, request)`